<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowRules</class>
 <widget class="QMainWindow" name="WindowRules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>470</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NumberPage">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PreviousPageButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Попередня</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextPageButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>470</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Наступна</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseRuleButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Повернутися назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
